Gems/Atom/Feature/Common/Assets/atom_feature_common_asset_files.cmake

```
@@ -387,9 +391,15 @@ set(FILES
     Shaders/PostProcessing/BloomCompositeCS.azsl
     Shaders/PostProcessing/BloomCompositeCS.shader
     Shaders/PostProcessing/BloomDownsampleCS.azsl
-    Shaders/PostProcessing/BloomDownsampleCS.shader
+    Shaders/PostProcessing/BloomDownsampleCS.shader    
```

nit: some white space snuck in here
```suggestion
    Shaders/PostProcessing/BloomDownsampleCS.shader
```